### PR TITLE
Update with 5.7 breaking changes

### PR DESCRIPTION
**Pull request by jsiemensLatitudegeo**

### CODE REVIEW

File: App1/App1/App1.csproj
Comment (Felicity-R) [inline App1/App1/App1.csproj:11]: So I suppose if another Workflow fix gets in we'll need to remember to update the workflow version here again as well.

**Review by Felicity-R — APPROVED** [review]

File: App1/App1.Android/App1.Android.csproj
Comment (ianschmitz) [inline App1/App1.Android/App1.Android.csproj]: This is required now i'm assuming?

File: App1/App1.Android/App1.Android.csproj
Comment (ianschmitz) [inline App1/App1.Android/App1.Android.csproj]: Do you mind adding back the newlines that were removed in these csproj files?

**Review by ianschmitz — APPROVED** [review]

**Review by kewalker — CHANGES_REQUESTED** [review]
There are a few paths in the `app.json` and `layout-*.xml` that need to be updated from 'ViewerFramework' to 'ViewerSpec'.

Also, the position of the the measurement tool in these layouts is `bottom-right` (with the geolocation components), whereas I think the default may be `bottom-left` now. Likely not required, but may be nice to be consistent.

One last thought is that we might opt to .gitignore this resource.designer file, since it seems to produce huge, unhelpful diffs:
`App1/App1.Android/Resources/Resource.designer.cs`
I think it's generated at build time and the analogous file is ignored in the GXM repo.

**Comment by Noah-Silvera** [comment]
I noticed when I was trying to use this branch locally that the `vertigis.mobile` package is not published on nuget,  so this branch only works if you have access to Latitude's package server. I think we should publish the `vertigis.mobile` package before merging this. Don't know if I'm preaching to the choir or not, but thought I'd call it out!

https://www.nuget.org/packages?q=vertigis

**Comment by jsiemensLatitudegeo** [comment]
> I noticed when I was trying to use this branch locally that the `vertigis.mobile` package is not published on nuget, so this branch only works if you have access to Latitude's package server. I think we should publish the `vertigis.mobile` package before merging this. Don't know if I'm preaching to the choir or not, but thought I'd call it out!
> 
> https://www.nuget.org/packages?q=vertigis

Absolutely. I won't merge this until the requisite nuget packages are all published to nuget.org.

File: App1/App1.Android/App1.Android.csproj
Comment (jsiemensLatitudegeo) [reply App1/App1.Android/App1.Android.csproj]: Ok..... Why?

File: App1/App1.Android/App1.Android.csproj
Comment (jsiemensLatitudegeo) [reply App1/App1.Android/App1.Android.csproj]: Good catch. That can be removed.

Comment (jsiemensLatitudegeo) [reply App1/App1.Android/App1.Android.csproj]: Removed.

File: App1/App1.Android/App1.Android.csproj
Comment (jsiemensLatitudegeo) [reply App1/App1.Android/App1.Android.csproj]: Done.

**Comment by jsiemensLatitudegeo** [comment]
> There are a few paths in the `app.json` and `layout-*.xml` that need to be updated from 'ViewerFramework' to 'ViewerSpec'.
> 
> Also, the position of the the measurement tool in these layouts is `bottom-right` (with the geolocation components), whereas I think the default may be `bottom-left` now. Likely not required, but may be nice to be consistent.
> 
> One last thought is that we might opt to .gitignore this resource.designer file, since it seems to produce huge, unhelpful diffs:
> `App1/App1.Android/Resources/Resource.designer.cs`
> I think it's generated at build time and the analogous file is ignored in the GXM repo.

All done.

File: App1/App1.Android/App1.Android.csproj
Comment (ianschmitz) [reply App1/App1.Android/App1.Android.csproj]: To avoid this:

image

As to why: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

File: .gitignore
Comment (ianschmitz) [inline .gitignore]: Can we ignore `Resource.designer.cs` generically? This ignore pattern will stop working as soon as someone renames their project (which we expected they will want to do).

**Review by Felicity-R — APPROVED** [review]

File: App1/App1.Android/App1.Android.csproj
Comment (ianschmitz) [reply App1/App1.Android/App1.Android.csproj]: There's still a number of occurrences in this PR.

Comment (jsiemensLatitudegeo) [reply App1/App1.Android/App1.Android.csproj]: Visual Studio seems to automatically remove those lines. I don't think it's feasible to try to go in and manually maintain this if Visual Studio is going to be constantly removing it. In any event, it seems like a rather arcane practice, for an OS that isn't supported (Linux and Unix). Visual Studio for Windows and Mac are the only two supported environments and they both work fine so I'm not sure what the issue is.

**Review by kewalker — APPROVED** [review]

File: .gitignore
Comment (jsiemensLatitudegeo) [reply .gitignore]: Done.

File: App1/App1.Android/App1.Android.csproj
Comment (ianschmitz) [reply App1/App1.Android/App1.Android.csproj]: That's fair.

Mac _is_ Unix based, and there's a Visual Studio `.editorconfig` setting it appears (`insert_final_newline`) to control this behavior, but I'll let you guys decide how you want to manage this.

Comment (jsiemensLatitudegeo) [reply App1/App1.Android/App1.Android.csproj]: I added in the newline at the end of the files and a .editorconfig file.

Comment (ianschmitz) [reply App1/App1.Android/App1.Android.csproj]: Beauty. Thanks!

**Pull request merged**
